### ISSUE TITLE
SUTD Climbers Freshmore Bouldering Introduction Session

### ISSUE THREAD

**Issue by SUTD-Climbers** [OpenSUTD/DiscoverSUTD-2021-microsite#101]
**Session Name**
SUTD Climbers Freshmore Bouldering Introducion Session

**Abstract**
To let freshmore students experience the sport of bouldering and interact with the current SUTD climb team

**Student Organisation organising this session**
SUTD Climbers

**Event Format**
Physical

**Session intent (what to expect)**
The freshmore climbing introduction session aims to introduce freshmore students to the sport of bouldering by letting them experience it first hand under the guidance of the senior members of the climbing team. Participants will be taught the basics of bouldering, i.e. _breaking falls, the different holds and the basics of bouldering_. Participants do not need to have any pre requisite knowledge in bouldering to join.

**Session details**
The session capacity will be 25 pax per session. We will be using the bouldering wall in SUTD and hope to have 4 sessions spread throughout the DiscoverSUTD weeks (15, 16, 22 & 23 Sep). The sessions can be held from 730pm to 930pm on weekdays except Monday. (Will update with the confirmed dates)

**Speakers/Workshop facilitators**
The facilitators of the activity will be the 7 current SUTD Climbers exco members, who have between 1 to 4 years worth of climbing experience.

**Sign up link(if any)**
https://forms.office.com/r/JKY0Z5t2zk

**Logistical details / Budget Needed**
Duct tape and permanent markers  - to help label routes set for the freshies to try ($30)
Club email: climbers@club.sutd.edu.sg

Help us with the overall resourcing of our events. This will give us a cue as to how to budget for the event and if there are further constraints we must consider.
Please provide us with your club e-mail address so we can notify you once your event has been scheduled. Copy this [template](https://forms.office.com/Pages/ShareFormPage.aspx?id=drd2NJDpck-5UGJImDFiPWH2rk0WPLpNoz88PwBGAxpUQ0c5TktQNlJKUTVMUFFQTVRHTkJOV1o1VS4u&sharetoken=zv60rxgRFw6Sp0GdVpWP) to ensure consistent branding throughout DiscoverSUTD.

**Comment by ExtremelySunnyYK** [comment]
hi @SUTD-Climbers! Thanks for submitting the github issue to us!

A few questions before we can proceed:
1. Could you provide the estimated budget required for Duct tape and permanent markers and approximately how many you would need?
2. Is there a sign up link for the event? It seems like it might have been missed out.

Thank you!

**Comment by castlemas** [comment]
Hi @SUTD-Climbers, 

May I clarify that the 25 pax per session means that it includes the number of facilitators too?
Would be great if you could let me know by today.

Thank you xx

**Comment by SUTD-Climbers** [comment]
1. 25 pax is the participant count. I believe the current limit of training sizes by OSL is 30 including coaches, with everyone split into groups of 5. Hence, 25 pax was set as the cap on the amount of participants we want to take per session. We will have at least 3 facilitators per session.
2. Noted, I will discuss with my exco and update here with the new date we have chosen.

**Comment by castlemas** [comment]
> 1. 25 pax is the participant count. I believe the current limit of training sizes by OSL is 30 including coaches, with everyone split into groups of 5. Hence, 25 pax was set as the cap on the amount of participants we want to take per session. We will have at least 3 facilitators per session.
> 2. Noted, I will discuss with my exco and update here with the new date we have chosen.

@SUTD-Climbers 
Note that the booking reflected on the book system timing does not tally with the time you stated here.
I have attached the booking scheldue below:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/71215246/131448975-2270e37a-e316-425e-92b9-51b7fb66a719.png)
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/71215246/131448988-6f195e88-cb50-4a06-b819-ac1a84f0a36d.png)
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/71215246/131448992-ca01639d-0c69-431a-b736-d99c135068a2.png)
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/71215246/131448997-3d8d32ea-69c6-4eb9-88a8-a69fcc779923.png)

**Comment by SUTD-Climbers** [comment]
For our session on 15 Sep is it possible to move it to 14 Sep since it clashes with the hostel event?
By default OSL has booked the wall for us as 2 separate sessions from 630pm-830pm and 840pm-1045pm so that we can accommodate 2 sessions in a day. However, we prefer to have only 1 session from 730pm-930pm within the duration of the 2 booked sessions so that we have buffer time before and after the session.

**Comment by castlemas** [comment]
@SUTD-Climbers 

Does this mean that your OSL would be helping with the booking of the venue for this event ?

**Comment by SUTD-Climbers** [comment]
@castlemas 

I got an email from OSL saying that they have booked the rock wall from 13 September 2021 to 23 January 2022 from Monday - Thursday 6.30pm - 8.30pm & 8.4pm - 10.45pm so I would assume that they have already done so.

**Comment by castlemas** [comment]
> @castlemas 
> 
> I got an email from OSL saying that they have booked the rock wall from 13 September 2021 to 23 January 2022 from Monday - Thursday 6.30pm - 8.30pm & 8.4pm - 10.45pm so I would assume that they have already done so.

@SUTD-Climbers 

Hi, I spoke to my OSL, we would not be able to hold on 14 Sept.

Would the team prehaps consider the third week instead 
28/9 or  28/9  730-930pm

Please refer to our timeline [here](https://opensutd.org/DiscoverSUTD-2021-microsite/proposal_details/) and choose a  date that is labeled Fifthrows session!


Thank you

**Comment by SUTD-Climbers** [comment]
@castlemas 

I think in that case we will just have 3 sessions. Thanks!

**Comment by makehahaz** [comment]
Hi @SUTD-Climbers . Just to update that your $30 budget request has been approved to get your materials. More details on the claim process will be let known
